contracts/Moderated.sol

```diff
@@ -9,20 +9,20 @@ import "@openzeppelin/contracts/ownership/Ownable.sol";
 contract Moderated is Ownable {
 
   // The address of the moderator
-  address public moderator;
+  mapping(address => bool) public moderators;
 
   modifier onlyModerator() {
-    require(msg.sender == moderator, "Only a moderator can do this");
+    require(moderators[msg.sender], "Only a moderator can do this");
     _;
   }
 
   constructor() public Ownable() {
-    moderator = owner();
+    moderators[owner()] = true;
```

Comment:
I don't know if this should be empty or not as assuming we are the owners of the contract originally, should we be moderators

Reply:
I think its a reasonable the owner starts off as a moderator

Reply:
especially since only a moderator can add another moderator

Reply:
i think it's ok. also it is easy to change in future.